stdlib/multiprocessing/context.pyi

```diff
@@ -27,11 +26,11 @@ class TimeoutError(ProcessError): ...
 class AuthenticationError(ProcessError): ...
 
 class BaseContext:
-    Process: type[BaseProcess]
-    ProcessError: type[Exception]
-    BufferTooShort: type[Exception]
-    TimeoutError: type[Exception]
-    AuthenticationError: type[Exception]
+    Process: ClassVar[type[BaseProcess]]
+    ProcessError: ClassVar[type[Exception]]
+    BufferTooShort: ClassVar[type[Exception]]
+    TimeoutError: ClassVar[type[Exception]]
+    AuthenticationError: ClassVar[type[Exception]]
```

Comment:
pyright is fine with this -- the pyright variable-scope bug only exists for assignments, not attribute annotations

```suggestion
    ProcessError: ClassVar[type[ProcessError]]
    BufferTooShort: ClassVar[type[BufferTooShort]]
    TimeoutError: ClassVar[type[TimeoutError]]
    AuthenticationError: ClassVar[type[AuthenticationError]]
```